exercises/concept/fruit-picker/.docs/hints.md

```diff
@@ -4,34 +4,26 @@
 
 - Callback functions are functions passed as an argument to a calling function.
 - Callback functions must meet the specification of the calling function.
-- These callbacks are all synchronous, meaning they all must return a value.
 
-## 1. Check if the grocer's service is online
+## 1. Try to notify with a successful message
 
-- Use the imported API function `checkStatus` in your function.
-- Pass a callback function to `checkStatus`. It should expect to receive a string argument.
-- Return the result from the `checkStatus` API function.
+- Use the imported API function `notify` in your function.
+- Pass an object to `notify`. It should expect to receive an object with a property of `message`. Think about what the message property should be.
+- Return the result from the `notify` API function.
 
-## 2. See if the grocer has some fruit
+## 2. Try to notify with an error message
 
-- Use the imported API function `checkInventory` in your function.
-- Create the query _object_ to pass as an argument.
-  - Follow the template in the instructions.
-- Pass along the callback function to the `checkInventory` API function.
-- Return the result from the `checkInventory` API function.
+- Use the imported API function `notify` in your function.
+- Pass another object to `notify`. It should expect to receive an object with a property of `message` like the previous task but with a different value.
+- Return the result from the `notify` API function.
```

Comment:
```suggestion
- Pay attention to the JSDoc type hint -- the function should not return a value.
```

See above

Reply:
My two cents on this is that maybe the wording of this can be changed to 'Pay attention to the comments above the function definition'. I (in my noobishness) think that 'JSDoc type' is kinda confusing. Otherwise, I would like to thank you for all of your comments I agree with everything.

Reply:
Yea, happy to iterate on it for sure! Maybe something like this?

```suggestion
- Pay attention to the comments [JSDoc] above the function definition:
  -  The function should not return a value.
```